src/parse.d

```diff
@@ -4087,10 +4087,16 @@ public:
                     pAttrs.storageClass &= (STCsafe | STCsystem | STCtrusted);
                 }
                 Dsymbol s = v;
+                if (structalign != 0)
```

Comment:
Right, so as the value of structalign should have been assigned correctly [here](https://github.com/D-Programming-Language/dmd/pull/5284/files#diff-e55efe6f4373bd0f2e2da9a75322a544R3762), shouldn't this check be ``structalign != STRUCTALIGN_DEFAULT`` ?